### ISSUE TITLE
Usage instructions causing bundler error

### ISSUE THREAD

**Issue by kyleect** [josh/sprockets-es6#17]
I added

    gem 'sprockets'
    gem 'sprockets-es6'

to my Gemfile and ran `bundle install` but it failed with this error:

    Bundler could not find comaptible versions for gem 'sprockets'
      In Gemfile:
        sprockets-es6 (<= 0) ruby depends on
          sprockets (= 3.0.0.beta.3) ruby
        sass-rails (~> 4.0.3) ruby depends on
          sprockets (2.8.0) ruby

**Comment by 007lva** [comment]
tries with gem 'sprockets', '~>3.0.0.beta' instead of gem 'sprockets'

**Comment by kyleect** [comment]
I edited my Gemfile with your suggestion and reran `bundle install` and received this error:

    Bundler could not find comaptible versions for gem 'sprockets'
      In Gemfile:
        sass-rails (~> 4.0.3) ruby depends on
          sprockets (<= 2.11.0, ~> 2.8) ruby
        sprockets (3.0.0.beta.1)

Sorry my experience with bundler and gem versions isn't strong so I don't know if this is something I'm doing wrong or something I can fix.

**Comment by hugeen** [comment]
update the version of sass-rails in your Gemfile

**Comment by kyleect** [comment]
I updated my sass-rails gem to 5.0.3 and ran `bundle install`. Received an error `Could not find gem 'sprockets, 3.0.0.beta.3 (>= 0) ruby' in the gems available on this machine`. After some googling I ran `gem install sprockets -v 3.0.0.beta.3` to try and fix the error. The 3.0.0.beta.3 gem installed fine so I ran `bundle install` and received the same error `Could not find gem 'sprockets, 3.0.0.beta.3 (>= 0) ruby' in the gems available on this machine`. Additional googling yielded the advice of using `bundle install --full-index` but it still spit out the same error.
